packages/varnish/src/components/logos/AI2Projects.tsx

```diff
@@ -1,7 +1,6 @@
 import * as React from 'react';
 
 export interface IconProps {
-  className?: string;
```

Comment:
why drop classname passthrough?

Reply:
hit an eslint error since this prop wasn't being used anywhere so i had to delete this to make yarn lint happy

Reply:
got it, sorry i didn't see those when i moved the svgs over since they were hard coded strings; i'll switch.